### PR TITLE
Feature/mainpage demo

### PR DESCRIPTION
**Pull request by madvingovich**

### CODE REVIEW

File: src/components/App.svelte [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
 
-<h1 class="text-red-500 text-5xl text-center font-hairline py-8">
-  Hello, it's me, {name}!
-</h1>
+<style>
+
+</style>
+
+<div class="container mx-auto p-3 sm:py-4">
+  <Header />
```
Comment (dimkabuki) [inline src/components/App.svelte:13]: I would destruct header component to smaller peaces. 
What the point to keep it together?
Header title it's just a text, random quote could be separate component and date is already dedicated

File: src/components/App.svelte [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+</div>
+<main>
+  <div class="lg:hidden py-8">
+    <Slider>
+      {#each [1, 2, 3] as number (number)}
```
Comment (dimkabuki) [inline src/components/App.svelte:18]: On preview it opens a second slide by default. Is it by design or it's a bug?

Comment (madvingovich) [reply src/components/App.svelte:18]: It's by design, activeSlide === 1 by default

Comment (dimkabuki) [reply src/components/App.svelte:18]: It's a bit strange.
When there's a list, I would expect the first item from the beginning

Comment (madvingovich) [reply src/components/App.svelte:33]: It was just for a test. I wanted immediately see that slide is on the screen center and I can see slides on the sides)

Comment (dimkabuki) [reply src/components/App.svelte:33]: For this purpose, I think storybook has some functionality, but not sure about svelte. With React has for sure

**Pull request merged**